src/services/ExperianService.ts

```diff
@@ -82,7 +82,8 @@ export class ExperianService {
     							},
     							names: [
     								{
-    									firstName: givenName,
+    									firstName: firstName,
+										middleNames: "",
```

Comment:
Rather than pass an empty string can we just exclude this field.

Reply:
Done